draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -4552,14 +4552,14 @@ RESPONSE:
         number for this document.]</t>
 
          <section anchor="IANA.CDNI.payload.ci-trigger-command.create.v2" title="CDNI ci-trigger-command.create.v2 Payload Type">
-          <t>Purpose: The purpose of this payload type is to distinguish version 2 of the CI/T trigger command object
+          <t>Purpose: The purpose of this payload type is to distinguish of the CI/T v2 create trigger command object
```

Comment:
"to distinguish of the CI/T" -> "to distinguish the CI/T" (extra "of" in the sentence)